src/main/java/com/amazon/opendistroforelasticsearch/security/dlic/rest/api/MigrateApiAction.java

```diff
@@ -76,8 +81,8 @@ public MigrateApiAction(final Settings settings, final Path configPath, final Re
     }
 
     @Override
-    protected void registerHandlers(RestController controller, Settings settings) {
-        controller.registerHandler(Method.POST, "/_opendistro/_security/api/migrate", this);
+    public List<Route> routes() {
```

Comment:
for my understanding why are we using routes for these APIs and not using registerHandlers?

Reply:
ES has changed the API in 7.7 to use routes() to register all the REST handlers. 

Here is the change by ES: https://github.com/elastic/elasticsearch/pull/51950

Reply:
Thanks for clarifying.